### DISCUSSION TITLE
How to increasre the   maximum number players in Practice Match's maps?

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1216] [Help and Support]
The official map like  **Fortitude**  allows  at most 6 players playingat the same time, but if I create a new map, I can't do it, only 4  players are allowed. How the official maps allow more than 4 players? I can't find options to increase it.

**Comment by qreeves** [comment]
Whilst in edit mode, you change the map variable `numplayers`. For example: `/numplayers 6` for 6 players. NOTE: You must be in edit mode and save the map afterwards for the changes to be permanent.

**Comment by SniperGoth** [comment]
Yes, as quin said, it's `numplayers`.

Or you can put more bots by typing  `addbot` , or increasing the `botoffset` to a higher number.

**Comment by ghost** [comment]
Thanks,but I have another qusetion: How can I find other commands like above? The Documentation don't have any information about it.
